home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -1171,6 +1173,53 @@ paths:
         default:
           $ref: '#/components/responses/InternalServerError'
 
+  /polls:
+    get:
+      tags:
+        - poll
+      summary: Get all polls
+      description: The endpoint gets all polls
+      operationId: queryPoll
+      parameters:
+        - $ref: '#/components/parameters/page_number'
+        - $ref: '#/components/parameters/page_size'
+        - $ref: '#/components/parameters/sort'
+        - $ref: '#/components/parameters/filter'
+        - $ref: '#/components/parameters/cooperation_id'
+        - $ref: '#/components/parameters/id'
+      responses:
```

Comment:
Could we add more query params for search like creation_date, completion_date, poll_type, poll_status like in '/cooperation/{cooperation_id}/polls' endpoint?

Reply:
Ok, I'll add it.